kolla/node_custom_config/ironic.conf

```diff
@@ -6,6 +6,9 @@ default_network_interface = neutron
 my_ip = "{{ kolla_internal_vip_address }}"
 enabled_storage_interfaces = cinder,noop
 default_storage_interface = cinder
+enabled_management_interfaces = ipmitool,noop
+default_management_interface = ipmitool
```

Comment:
FYI the first enabled iface will be the default, according to the docs. Good to be explicit though.